docs/rules/object-curly-newline.md

```diff
@@ -19,6 +19,7 @@ Or an object option:
 
 * `"multiline": true` (default) requires line breaks if there are line breaks inside properties or between properties
 * `"minProperties"` requires line breaks if the number of properties is more than the given integer
+* `"consistent"` requires that either both curly braces, or neither, directly enclose newlines
```

Comment:
Sorry, I made a typo.

I meant to say:

Nit: Can you describe this as `"consistent": true` instead of just `"consistent"` so it's clearer that this is a boolean?